packages/f/fx-gltf/xmake.lua

```diff
@@ -0,0 +1,24 @@
+package("fx-gltf")
+    set_homepage("https://github.com/jessey-git/fx-gltf")
+    set_description("A C++14/C++17 header-only library for simple, efficient, and robust serialization/deserialization of glTF 2.0")
+    set_license("MIT")
+
+    add_urls("https://github.com/jessey-git/fx-gltf/archive/refs/tags/v$(version).tar.gz", {alias = "archive"})
+    add_urls("https://github.com/jessey-git/fx-gltf.git", {alias = "github"})
+
+    add_versions("github:1.2.0", "v1.2.0")
+    add_versions("archive:1.2.0", "d8eaceba72ea6574b982c7b0d2328fd3f8ad519db4a37cf63cd3f8020d7722bf")
```

Comment:
we need only the following line.

add_versions("v1.2.0", "d8eaceba72ea6574b982c7b0d2328fd3f8ad519db4a37cf63cd3f8020d7722bf")

xmake will checkout v1.2.0 tag if git url is added and tag exists

Reply:
like this https://github.com/xmake-io/xmake-repo/blob/3dafd241e265fcc3b7282d9b7ca1b2aeff83d4f6/packages/t/tbox/xmake.lua#L11

Reply:
should i remove the aliases then?

Reply:
yeah, and remove `v` prefix of $(version) in url